quickstart/app/Http/Requests/Category/UpdateRequest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Requests\Category;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'cat_name' => 'required|unique:categories,cat_name,'.request()->id ,
+            'cat_desc' => 'required'
```

Comment:
```suggestion
            'cat_desc' => 'required',
```
thêm dấu , vào phần tử cuối cùng của mảng